src/orga/minutes/2020-09-02.md

```diff
@@ -0,0 +1,178 @@
+# 2020-09-02 conda-forge core meeting 
+
+****
+
+[Zoom link](https://flatiron.zoom.us/j/93242638216?pwd=bjRCWmVJRW1oTGJhN09VUmxtTTJOUT09 )
+[What time is the meeting in my time zone](https://arewemeetingyet.com/UTC/2020-08-26/17:00/w/Conda-forge%20dev%20meeting#eyJ1cmwiOiJodHRwczovL2hhY2ttZC5pby9wUk15dFVKV1FmU3NJM2xvMGlqQzJRP2VkaXQifQ==)
+[last weeks meeting](https://hackmd.io/uoxd_bzSRie5Dq1ZrMSvhA)
+
+## Attendees
+
+- Matt B.
```

Comment:
A lot was done in a meeting with only one person attending ;-p

Reply:
You know how I roll...